app/build.gradle

```diff
@@ -72,3 +82,11 @@ android {
         exclude("META-INF/*.kotlin_module")
     }
 }
+
+kapt {
```

Comment:
You don't need to update it now, but I think it is good to you know that KAPT will eventually be deprecated in favor of KSP. [Kotlin Symbol Processing](https://android-developers.googleblog.com/2021/09/accelerated-kotlin-build-times-with.html) is a much better solution related to build times and probably in more meaningful error messages as well. I didn't try it yet and believe that Dagger/Hilt support is experimental at the moment, but keep in touch!